src/Logger.cc

```diff
@@ -0,0 +1,87 @@
+/* This file is part of the CARTA Image Viewer: https://github.com/CARTAvis/carta-backend
+   Copyright 2018, 2019, 2020, 2021 Academia Sinica Institute of Astronomy and Astrophysics (ASIAA),
+   Associated Universities, Inc. (AUI) and the Inter-University Institute for Data Intensive Astronomy (IDIA)
+   SPDX-License-Identifier: GPL-3.0-or-later
+*/
+
+#include "Logger.h"
```

Comment:
I think we should move these files into a `Logger` folder. I also think the function definitions in `Logger.h` should be moved into `Logger.tcc`